src/js/components/Form/stories/Components.js

```diff
@@ -17,8 +18,8 @@ import { grommet } from 'grommet/themes';
 
 const Example = () => (
   <Grommet full theme={grommet}>
-    <Box fill align="center" justify="center">
-      <Box width="medium">
+    <Box fill overflow="auto" align="center" justify="center" pad="large">
```

Comment:
In this Example story, I've dragged a screenshot to the FileInput, and removed it. 
As a result, I've noticed a few things in the context of Form:
![image](https://user-images.githubusercontent.com/6320236/95128981-7c8a1b00-0717-11eb-8c65-6af05ab7aa33.png)

1. The border-color remains red, which I think should go back to the default border-color.
2. The FormField bottom border is colored with the focus color, but the focus color should be placed more clearly on the FileInut component itself.

Reply:
I've fixed the lingering red border color.

The FormField focus decoration is how the grommet theme indicates focus for all input types.